wsgi_kerberos.py

```diff
@@ -77,10 +91,21 @@ class KerberosAuthMiddleware(object):
     :param auth_required_callback: predicate accepting the WSGI environ
         for a request returning whether the request should be authenticated
     :type auth_required_callback: callable
+    :param read_max_on_401: When a request could not be authenticated, read and
```

Comment:
This technically applies to 401 and 403. What do you think of `read_max_on_auth_fail`?

Reply:
Done in the latest revision.